### ISSUE TITLE
Can't start - load not a function?

### ISSUE THREAD

**Issue by seajaysec** [JaneJeon/Fix-Shitty-Wifi#6]
Installed fine with npm. When starting up, I get this error:

```
sudo node index.js
/Users/[REDACTED]/githubclones/Fix-Shitty-Wifi/index.js:1
require("dotenv").load()
                  ^

TypeError: require(...).load is not a function
    at Object.<anonymous> (/Users/[REDACTED]/githubclones/Fix-Shitty-Wifi/index.js:1:19)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

**Comment by squarerobin** [comment]
Hello, I've created a PR that might fix this: https://github.com/JaneJeon/Fix-Shitty-Wifi/pull/8

**Comment by JaneJeon** [comment]
@seajaysec wow, I haven't bee notified of this. I'll push a simple fix - #8 is extremely overkill.

**Comment by JaneJeon** [comment]
@seajaysec should be working now! Make sure to `yarn/npm install` before starting up!
